src/main/resources/db/changelog/changeset/0011_agency_demographics_gender_column_change/genderColumn-rename.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE `agencyservice`.`agency`
+ALTER TABLE `agencyservice`.`agency` RENAME COLUMN genders TO gender;
```

Comment:
I think the rename should rename to "genders"... 
and the fallback should go back to "gender" isn't it?
first line is copy past error?